src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -120,42 +122,90 @@ function organizationDashboard(): JSX.Element {
               </Row>
             ) : (
               <Row style={{ display: 'flex' }}>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/orgpeople/id=${organizationId}`);
+                  }}
+                >
                   <DashBoardCard
                     count={data?.organizations[0].members?.length}
                     title={t('members')}
                     icon={<UsersIcon fill="var(--bs-primary)" />}
                   />
                 </Col>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/orgpeople/id=${organizationId}`);
+                  }}
+                >
                   <DashBoardCard
                     count={data?.organizations[0].admins?.length}
                     title={t('admins')}
                     icon={<AdminsIcon fill="var(--bs-primary)" />}
                   />
                 </Col>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/orgpost/id=${organizationId}`);
+                  }}
+                >
                   <DashBoardCard
                     count={postData?.postsByOrganization?.length}
                     title={t('posts')}
                     icon={<PostsIcon fill="var(--bs-primary)" />}
                   />
                 </Col>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/orgevents/id=${organizationId}`);
+                  }}
+                >
                   <DashBoardCard
                     count={eventData?.eventsByOrganization?.length}
                     title={t('events')}
                     icon={<EventsIcon fill="var(--bs-primary)" />}
                   />
                 </Col>
-                <Col xs={6} sm={4} className="mb-4">
+                <Col
+                  xs={6}
+                  sm={4}
+                  role="button"
+                  className="mb-4"
+                  onClick={(): void => {
+                    history.push(`/blockuser/id=${organizationId}`);
```

Comment:
Similarly create a variable for this too.

Reply:
I added the variable for this.